authors/apps/articles/views.py

```diff
@@ -57,3 +61,79 @@ def destroy(self, request, pk, **kwargs):
                 return Response("You have succesfully deleted the article")
             return Response("You do not have permision to delete the article")
         return Response(self.err_message, status=status.HTTP_404_NOT_FOUND)
+
+
+class Like(APIView):
+    """
+    POST articles/:id/like/ to like an article
+    GET articles/:id/ and view likes_count
+    """
+    permission_classes = (IsAuthenticated, )
+
+    def get_article(self, article_id):
+        """ Method to search for an article object by its id"""
+        try:
+            return Article.objects.get(id=article_id)
+        except Article.DoesNotExist:
+            raise Http404("Article does not exist")
```

Comment:
@KabohaJeanMark , this response is not caught. Try using `NotFound` from `rest_framework.exceptions`.